src/test/util_test.ts

```diff
@@ -0,0 +1,211 @@
+/* eslint-env mocha */
+import * as assert from 'assert';
+const espree = require('espree');
```

Comment:
no require consts. if its an old school commonjs export (`export = foo`), you need to `import espree = require('espree');`

Reply:
which also means you need to make sure you have types for these imports

Reply:
Hmmm I think I copied this from somewhere, will update